src/QGLView.cc

```diff
@@ -178,6 +178,13 @@ void QGLView::paintGL()
 {
   GLView::paintGL();
 
+#ifdef USE_QOPENGLWIDGET
+  if (mouseDoubleClicked != nullptr) {
+    centerCameraOnMouse(mouseDoubleClicked);
+    mouseDoubleClicked = nullptr;
+  }
+#endif
+
```

Comment:
You can simplify `if (mouseDoubleClicked != nullptr) {` to `if (mouseDoubleClicked) {`

I quote from https://stackoverflow.com/questions/11279715/nullptr-and-checking-if-a-pointer-points-to-a-valid-object
> ... So nullptr will be properly and implicitly cast to the boolean false so long as the compiler follows the C++11 language specification. ...

For double checking: https://www.ibm.com/support/knowledgecenter/en/ssw_ibm_i_73/rzarg/nullpt.htm
```
char* p = nullptr;     // p has the null pointer value 
...
if( p );               // evaluates to false
...
if( nullptr );         // OK, conversion to the bool type 
...
```

You also see it in practice in https://www.learncpp.com/cpp-tutorial/6-7a-null-pointers/ (and in some segments of OpenSCAD)